modules/serverless-functions-adding-annotations.adoc

```diff
@@ -0,0 +1,23 @@
+[id="serverless-functions-adding-annotations_{context}"]
+= Adding annotations to a function
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For every annotation that you want to add, add the following YAML to the `annotations` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+annotations:
+  <annotation_name>: "<annotation_value>" <1>
+----
+<1> Substitute `<annotation_name>: "<annotation_value>"` with your annotation.
+
```

Comment:
Hello @rh-max . I think some concrete example would be handy, just as Ashleigh and Lance already suggested. wdyt?

Reply:
You are right, I've added @lance 's example with Alice.